packages/shared/trade/src/asset-input/asset-input.tsx

```diff
@@ -142,7 +143,7 @@ export const AssetInput = React.forwardRef<
     if (!availableTokens) return <PageLoading />;
 
     return (
-      <Box>
+      <Box sx={{ marginBottom: `${theme.spacing(4)} !important` }}>
```

Comment:
What was this fix for?

Reply:
NOTE* the !importants are to prevent jumping when the errors appear and disappear. We still want the rest of the components in the sidebar to have the same margin bottom

Reply:
ok